src/resources/postcss/rsvp/_attendee-registration.pcss

```diff
@@ -115,17 +115,25 @@
 
 	.tribe-tickets__rsvp-ar-guest-list {
 		margin-top: var(--spacer-7) !important; /* @todo @juanfra: see why common is setting specificity for this. */
+		counter-reset: section;
 	}
 
 	.tribe-tickets__rsvp-ar-guest-list-item {
 		display: inline-block;
+		counter-increment: section;
 
 		@media(--viewport-medium) {
 			display: block;
 		}
 
 		&:not(:first-of-type) {
 
+			.tribe-tickets__rsvp-ar-guest-list-item-title {
+				&::after {
+					content: " " counter(section);
```

Comment:
this will only work for those languages that adds the guest number after the word "guest". If a language has the number in the middle of words (Chinese for example, 第2位客人) or at the beginning this will not work.

Instead, it would be better to see if we can get a translated string from the BE with a placeholder, pass it to the FE as well as the window object, and use it in the template and JS.

Reply:
I though of that at first, but the number is dynamic. So the translated string should be a "template" and I didn't find a way to do that, because using `%s` or whatever would be interpreted by gettext. How do you think we could use a translatable template?